nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -58,6 +58,7 @@ def __init__(self,
                  ignored_scopes: Optional[IgnoredScope] = None,
                  model_type: Optional[ModelType] = None,
                  fast_bias_correction: bool = True,
+                 inplace_statistics: bool = False,
```

Comment:
Here we have default value False, why don't we have it for MinMax?

Reply:
See https://github.com/openvinotoolkit/nncf/pull/1637#discussion_r1165360960